src/components/action-button.js

```diff
@@ -17,6 +17,7 @@ export default function renderButton({ layout, theme, app, constraints, senseNav
   const isClickable = !disabled && !constraints.active;
   const formattedStyles = styleFormatter.getStyles({ style, disabled, theme, element });
   button.setAttribute('style', formattedStyles);
+  button.setAttribute('tabindex', '-1');
```

Comment:
I know this will work, but it feel weird to remove the tabindex from a button. We might instead need to accept that it is not a normal button